ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -307,13 +308,15 @@ public QueryMetadata getQueryExecutionPlan(final Query query) throws Exception {
           new RegisterTopicCommand(
               (RegisterTopic) statement
           ),
-          tempMetaStoreForParser
+          tempMetaStoreForParser,
+          false
       );
       ddlCommandExec.tryExecute(
           new RegisterTopicCommand(
               (RegisterTopic) statement
           ),
-          tempMetaStore
+          tempMetaStore,
+          false
```

Comment:
This should be true here right? This is called from the validation phase, before the query makes it onto the command topic.

Reply:
Ignore this copy of the comment. I mean't to write it for line 379